README.md

```diff
@@ -34,12 +34,22 @@ The operator will then create a CatalogSourceConfig CR which will, for the time
 
```

Comment:
Please mention that the order is important

Reply:
Good idea, I have mentioned that order is important in the latest commit.